projects/admin/src/app/acquisition/components/account/account-list/account-list.component.ts

```diff
@@ -31,30 +35,61 @@ export class AccountListComponent implements OnInit {
   /** Root account to display */
   rootAccounts: IAcqAccount[] = [];
 
+  /** Export options configuration. */
+  exportOptions: {
+    label: string,
+    url: string,
+    disabled?: boolean,
+    disabled_message?: string
+  }[];
+
+  /** Store library pid */
+  private _libraryPid: string;
+
 
   // GETTER & SETTER ============================================================
   /** Get the current organisation */
   get organisation(): any {
     return this._organisationService.organisation;
   }
 
+  /**
+   * Return a message containing the reasons why record list cannot be exported
+   * @return Boolean
```

Comment:
Bad return statement ; should be 'string'. In general, for getter functions, we skip the return docstring line

Reply:
ok